### ISSUE TITLE
[SF] [ViewSettingsDialog] [a11y] ViewSettingsDialog accessibility issues

### ISSUE THREAD

**Issue by Neeeko** [SAP/ui5-webcomponents#4660]
### **Bug Description**
We have found three accessibility defects regarding the ViewSettingsDialog header:
1. The Title is not announced by the Screen Reader when opening the ViewSettingsDialog (tested with NVDA on Windows 10)
![Screenshot 2022-01-27 at 18 15 08](https://user-images.githubusercontent.com/7316793/151411845-f2240942-0934-4771-9a21-e099950c03b8.png)

2. The Title text is a `ui5-label` instead of `ui5-title` with heading level 2 (h2)
![Screenshot 2022-01-27 at 18 27 18](https://user-images.githubusercontent.com/7316793/151535560-b58e32b8-c5e0-488f-bf2a-22e6197fdd4e.png)

3. When opening the ViewSettingDialog, the focus will go to the header toggle button (which is fine), but then pressing TAB will move the focus to `Ascending` instead of `Sort Order`

### **Expected Behavior**
1. Dialog title should be announced by the screen reader

2. Heading levels should be consistent

3. When navigating using keyboard, GroupHeaders shouldn't be skipped

### **Steps to Reproduce**
These three issues can be found and reproduced using the [ViewSettingsDialog example](https://sap.github.io/ui5-webcomponents/playground/components/ViewSettingsDialog/).

### **Context**
- UI5 Web Components version: 1.0.1
- OS/Platform: Windows 10 (using NVDA as screen reader)
- Browser: Google Chrome
- Affected component: ViewSettingsDialog

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

### **Stakeholder Info** (if applicable)

- Organization: SuccessFactors

**Comment by rchrdchn** [comment]
Hey team – just following up on this issue. When could we expect the fix for this issue?

**Comment by tsanislavgatev** [comment]
Hello,

We are currently checking the issues. As they are 3, they might be fixed as single ones and one by one, not all of them together.

Best Regards,
Tsanislav

**Comment by rchrdchn** [comment]
Thanks! Please keep up posted.

On February 18, 2022, GitHub ***@***.***> wrote:
> Hello,
>
> We are currently checking the issues. As they are 3, they might be
> fixed as single ones and one by one, not all of them together.
>
> Best Regards,
>  Tsanislav
>

—
Reply to this email directly, view it on GitHub
<https://github.com/SAP/ui5-webcomponents/issues/4660#issuecomment-
1044209445>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/ACYKLVOBJBKH2VT64GTZBTTU3YGP7ANCNFSM5NAOGGWA>.
You are receiving this because you commented.Message ID: <SAP/ui5-
***@***.***>

**Comment by tsanislavgatev** [comment]
Hello @Neeeko @rchrdchn,

All of the issues are now fixed and merged into the master branch.

Best Regards,
Tsanislav

**Comment by Neeeko** [comment]
@tsanislavgatev, thank you! The first PR (https://github.com/SAP/ui5-webcomponents/pull/4783) seems to be released but not the second one (https://github.com/SAP/ui5-webcomponents/pull/4827). Any idea when it might be?
